extensions/kind/src/cli-run.ts

```
@@ -0,0 +1,103 @@
+  let command: string | undefined;
+  if (system === 'win32') {
+    // admin privileges are are not needed on Windows
+    await fs.promises.copyFile(binaryPath, destinationPath);
+    return destinationPath;
+  } else if (system === 'darwin') {
+    command = 'exec';
+    args = ['cp', '-f', binaryPath, destinationPath];
+  } else if (system === 'linux') {
+    command = '/bin/sh';
+    args = ['-c', `cp ${binaryPath} ${destinationPath}`];
+  }
```

Biome (2.5.3)

[error] This else clause can be omitted because previous branches break early.

(lint/style/noUselessElse)